Dockerfile

```diff
@@ -223,7 +223,8 @@ RUN source .venv/bin/activate && \
       timescaledb && \
     rm -rf /tmp/localstack/* && \
     rm -rf /var/lib/localstack/cache/* && \
-    chmod -R 777 /var/lib/localstack
+    chown localstack:localstack /usr/lib/localstack && \
```

Comment:
Just wondering - should this be a recursive `chown`?

Reply:
Good catch, thanks!